### DISCUSSION TITLE
Amazon.Lambda.Logging.AspNetCore vs. AWS.Logger.AspNetCore

### DISCUSSION THREAD

**Discussion by schmallaria** [aws/aws-lambda-dotnet#1305] [Q&A] [answered]
I have hosted an asp.net core 6 minimal API as docker container in an AWS lambda function using the [hosting package](https://github.com/aws/aws-lambda-dotnet/tree/master/Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting). 

What is the correct logging package to log message from my API to AWS CloudWatch?

[Amazon.Lambda.Logging.AspNetCore](https://github.com/aws/aws-lambda-dotnet/tree/master/Libraries/src/Amazon.Lambda.Logging.AspNetCore) or [AWS.Logger.AspNetCore](https://github.com/aws/aws-logging-dotnet)?

**Comment by ashishdhingra** [comment, marked as answer]
@schmallaria Since you would be using logging in the Lambda environment, it is recommended to use [Amazon.Lambda.Logging.AspNetCore](https://github.com/aws/aws-lambda-dotnet/tree/master/Libraries/src/Amazon.Lambda.Logging.AspNetCore). The guidance is mentioned at [https://github.com/aws/aws-logging-dotnet#aws-logging-net where it is mentioned](https://github.com/aws/aws-logging-dotnet#aws-lambda).

Thanks,
Ashish
